internal/cli/quickstarts.go

```diff
@@ -31,15 +40,21 @@ var (
 		}
 		return
 	}()
-)
 
-// QuickStart app types and defaults
-const (
-	qsNative    = "native"
-	qsSpa       = "spa"
-	qsWebApp    = "webapp"
-	qsBackend   = "backend"
-	_defaultURL = "http://localhost:3000"
+	clientID = Flag{
+		Name:       "ClientID",
```

Comment:
```suggestion
		Name:       "Client ID",
```
The name is used to build the prompt (eg: `Client ID:`).

Reply:
got it, thanks